specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/networkData.json

```diff
@@ -0,0 +1,361 @@
+      "description": "Network data on a resource",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/NetworkDataProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/types.json#/definitions/Resource"
+        }
+      ]
+    },
+    "NetworkDataProperties": {
+      "type": "object",
+      "description": "Describes the properties of the network data.",
+      "properties": {
+        "connectableResources": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/NetworkDataConnectableResource"
+          }
+        }
+      }
+    },
+    "NetworkDataConnectableResource": {
+      "type": "object",
+      "description": "Describes the allowed inbound and outbound traffic of an Azure resource",
+      "properties": {
+        "connectableResourceId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The connectable resource id"
+        },
+        "connectionType": {
+          "type": "string",
+          "description": "The connection type between the extension resource to the connectable resource Id",
+          "enum": [
+            "Azure vm"
+          ],
+          "x-ms-enum": {
+            "name": "connectionTypeValues",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "Azure vm"
+              }
+            ]
+          }
+        },
+        "allowedConnections": {
+          "readOnly": true,
+          "type": "object",
+          "description": "The allowed connections details",
+          "properties": {
+            "calculatedDateTime": {
+              "type": "string",
+              "format": "date-time",
+              "readOnly": true,
+              "description": "The UTC time on which the allowed connections was calculated"
+            },
+            "inboundConnectedResources": {
+              "$ref": "#/definitions/ConnectedResources"
+            },
+            "outboundConnectedResources": {
+              "$ref": "#/definitions/ConnectedResources"
+            }
+          }
+        },
+        "actualTraffic": {
+          "readOnly": true,
+          "type": "object",
+          "description": "The actual traffic details",
+          "properties": {
+            "startTime": {
+              "type": "string",
+              "format": "date-time",
+              "readOnly": true,
+              "description": "The UTC start time of the actual traffic time range"
+            },
+            "endTime": {
+              "type": "string",
+              "format": "date-time",
+              "readOnly": true,
+              "description": "The UTC end time of the actual traffic time range"
+            },
+            "trafficDataState": {
+              "type": "string",
+              "description": "Indicate whether the Azure monitor agent is installed on the vm",
```

Comment:
>ent is installed on the vm [](start = 67, length = 26)

will this be "Not Available" when you have connectionTypes that aren't "Azure VM"?  Or will a VM always be in the picture somehow?

Reply:
This will be N/A when the agent is not installed on the VM (user didn;t want it on the VM, installation failed for some reason, etc.)

Reply:
My question is about the relationship between this and the "connectionType" property.  When you introduce more "connectionTypes" won't the relevant data here change?  trafficDataState can't be Available if there is no VM in the picture

Reply:
That is not accurate - Since this API talks about Azure resources we will introduce only resources that has traffic data report capabilities. For example, Scale set, web sites, etc.
I agree that the description should be more generic and not say specifically Azure monitor agent or vm - We will change it.

Reply:
fixed.